docs/Rapid/3-User Manual/2-Explorer/1-Tables and items/data-tables/1-viewing-data-using-tables.md

```diff
@@ -69,7 +69,7 @@ If you would like to undo a quick filter and view all of the items again, there
 ## Data Tables can be Refreshed
 If the data in a table is updated, the table will need to be refreshed in order for that data to be visible in the table. Refreshing the entire page, as mentioned earlier, is not desirable as you may need to reauthenticate, which will take time.
 
-Rapid Platform includes an *Inline Refresh* button, that allows you to refresh all the items in a data table, without leaving the page. The *Inline Refresh* is located in the <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/User%20Manual/glossary#command-bar">Command Bar</a>, and has an icon that resembles a clockwise arrow.
+Rapid Platform includes an *Inline Refresh* button, that allows you to refresh all the items in a data table, without leaving the page. The *Inline Refresh* is located in the <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/User%20Manual/glossary#command-bar" target="_blank">Command Bar</a>, and has an icon that resembles a clockwise arrow.
```

Comment:
We can point links to `https://docs.rapidplatform.com/...` now as the domain has shifted